apps/builder/app/builder/features/ai/ai-fetch-result.ts

```diff
@@ -30,7 +30,7 @@ import {
 } from "~/shared/nano-states";
 import { applyOperations, patchTextInstance } from "./apply-operations";
 import { restAi } from "~/shared/router-utils";
-import { RequestParamsSchema } from "~/routes/rest.ai";
+import { RequestParamsSchema } from "~/routes/rest.ai._index";
```

Comment:
needs comment that this is a workaround?

Reply:
Basically don't
https://remix.run/docs/en/main/file-conventions/routes